nvflare/fuel/hci/server/login.py

```diff
@@ -176,8 +176,8 @@ def handle_logout(self, conn: Connection, args: List[str]):
         conn.append_string("OK")
 
     def pre_command(self, conn: Connection, args: List[str]):
-        if args[0] in [LOGIN_CMD_NAME, CERT_LOGIN_CMD_NAME]:
-            # skip login command
+        if args[0] in [LOGIN_CMD_NAME, CERT_LOGIN_CMD_NAME, CHECK_SESSION_CMD_NAME, LIST_SESSIONS_CMD_NAME]:
```

Comment:
I don't think the LIST_SESSIONS_CMD_NAME should be included here, because this command is issued by the user himself. This means that the user is still active, and the "last_active_time" should be updated.